Detectors/TPC/workflow/src/ChunkedDigitPublisher.cxx

```diff
@@ -19,10 +19,12 @@
 #include "Framework/DataAllocator.h"
 #include "Framework/ControlService.h"
 #include "DataFormatsTPC/Digit.h"
+#include "DataFormatsTPC/TPCSectorHeader.h"
 #include "CommonUtils/ConfigurableParam.h"
 #include "DetectorsRaw/HBFUtilsInitializer.h"
 #include "TPCSimulation/CommonMode.h"
 #include "DetectorsBase/Detector.h"
+#include "DPLUtils/MakeRootTreeWriterSpec.h"
```

Comment:
In the normal TPC digitizer workflow, we employ a specialized header `TPCDigitRootWriterSpec.h` which defines a Writer class for TPC digits. I would try to utilize this. Likely it offers correct treatment for the MC truth classes.